### ISSUE TITLE
Wrong Interpolation Syntax

### ISSUE THREAD

**Issue by geckoslair** [mike-goodwin/pug-bootstrap#5]
Hi Mike,

i'm starting using your mixins library. I've noticed that the interpolation syntax is not compatible with Pug 2. For examples:

`mixin nav(type,items,active)`
`ul.nav(class="#{type}")`

in [https://github.com/mike-goodwin/pug-bootstrap/blob/master/components/navs.pug](url).

Spec. Ref: [https://pugjs.org/api/migration-v2.html](url)

are you going to update your library?
Thank you for the support and the usefull porting.
Enrico.

**Comment by mike-goodwin-sage** [comment]
Hello Enrico - Yes, I only migrated the specific parts of the library that I was using for my own project, I'm sorry about that :(

I did consider migrating it all, but I was trying to ensure that there are unit tests for any change that I make and some of the mixins from the original jade-bootstrap project do not have tests in place yet. If the nav mixin the only one you are interested in? If so I could look at migrating that part of it.

Alternatively, I would be very happy to accept any pull requests :)
